application/core/plugins/Authdb/Authdb.php

```diff
@@ -62,11 +62,12 @@ public function createNewUser()
             $new_email = flattenText($preCollectedUserArray['email']);
             $new_full_name = flattenText($preCollectedUserArray['full_name']);
             $presetPassword = flattenText($preCollectedUserArray['password']);
+            $status = $preCollectedUserArray['status'];
```

Comment:
It's not enough to change only the Authdb plugin, because there are many other auth plugins, e.g. SAML. The limitation to block non-activated users from logging in must happen somewhere else.

Reply:
new login with low priority (happen after all) at https://manual.limesurvey.org/AfterSuccessfulLogin  ?

I think it can work.

Reply:
AfterSuccessfulLogin will not be useful in this context.

This functionality is intended to create a user with a predetermined status based on your chosen plan. If the maximum number of active users is reached, any additional accounts will be automatically deactivated; otherwise, they will be activated.

Reply:
Moving it to the LS Cloud repository, it will always remain active for the CE.

Reply:
> AfterSuccessfulLogin will not be useful in this context.

We don't know if it's a new user here ? 

Maybe need to add an event inside Auth(DB|LDAP|…) plugin ?

Reply:
We will use createNewUser but will move it to LS cloud instead.